packages/twenty-front/src/utils/date-utils.ts

```diff
@@ -122,11 +124,12 @@ export const beautifyDateDiff = (
     ['years', 'days'],
   );
   let result = '';
-  if (dateDiff.years) result = result + `${dateDiff.years} year`;
+  if (isTruthy(dateDiff.years)) result = result + `${dateDiff.years} year`;
```

Comment:
You can disable the rule here and add a TODO

Reply:
Alright!